ultralist/app.go

```diff
@@ -389,6 +329,94 @@ func (a *App) Sync(quiet bool) {
 	}
 }
 
+// SetupSync sets up a todolist to sync with ultralist.io.
+func (a *App) SetupSync() {
+	backend := NewBackend()
+	if !backend.CredsFileExists() {
+		fmt.Println("You're not authenticated with ultralist.io yet.  Please run `ultralist auth` first.")
```

Comment:
this error isnt strictly true, it is only checking for if the file exists and could contain bad auth keys.

NewBackend(), already check and loads this file, so would a more correct test be.

```
    if backend.Credentials == "" {
		fmt.Println("You're not authenticated with ultralist.io yet.  Please run `ultralist auth` first.")
                 return
}
```

Side note: If you are using JWT (I dont have a UL account so I  am guessing) as the authentication, should we also check to see it is `valid` before trying to use it? or as part of the loadCreds, or a new function checkCreds.

Reply:
ah, that's a good catch!

Regarding the JWT, I don't feel strongly about validating, since the request would ultimately fail if they have invalid creds.

Reply:
I did try it with just using `backend.Creds`, and it didn't seem to work.  idk :shrug:

Reply:
try with a ~/.config/ultralist/cred.json with `{}`.

Also a blank file will cause a panic when NewBackend() tries to parse it.

I was thinking about turning those panics into a log and non-0 exit.

Reply:
I'm not sure there's a scenario where creds.json would have that info in it though.